website/docs/commands/fmt.md

```diff
@@ -20,29 +20,26 @@ You can check formatting correctness using a `--check` flag:
 scala-cli fmt --check
 ```
 
-### Dialects
+### Scalafmt version and dialect
 
-Scala CLI also supports dialects that are passed to the formatter.
-This value is only used if there is no `.scalafmt.conf` file.
-However, if it exists, then all configuration should be placed there.
-For a list of all possible values, consult
-the [official Scala Dialects documentation](https://scalameta.org/scalafmt/docs/configuration.html#scala-dialects):
+Configuration file with specified **version** and **dialect** is required to run `scalafmt`. Scala CLI `fmt` tries to find the `.scalafmt.conf` file in the **projects root** directory and **current workspace** directory. There are 3 possible cases:
```

Comment:
Actually, the configuration file is no longer required, the way I understand it.
Instead:
```suggestion
Specifying the  **version** and **dialect** is required to run `scalafmt`. Those can be passed through the `scalafmt` configuration file or directly with Scala CLI. 

Scala CLI's `fmt` sub-command tries to find the `.scalafmt.conf` file in the **projects root** directory and **current workspace** directory. 

There are 3 possible cases:
```

Reply:
Scala CLI `fmt` command runs `scalafmt` under the hood and as far as I know `scalafmt` still needs `.scalafmt.conf` file.

Reply:
Maybe I should start with information that `fmt` command runs `scalafmt` under the hood, but this is how description of [fmt](https://scala-cli.virtuslab.org/docs/commands/fmt) starts.

Reply:
we possibly could also clarify what do we mean by the project root here (since here we calculate it based on the existing `git` configuration, and not based on the directory of the first input, like in other sub-commands)

Reply:
> Scala CLI fmt command runs scalafmt under the hood and as far as I know scalafmt still needs .scalafmt.conf file.
Ok, fair point.

What I mean is - from the user's perspective, the important part is the `fmt` sub-command, which now does not need the conf file. 

You're right that we can and should clarify how we do it under the hood (since one can peek into the `.scala-build` folder on how we do it), but IMO the reader of this doc should be clear that when using the `fmt` command, he doesn't need to specify anything, really (as long as he's fine with the defaults we infer)

Reply:
Sounds good. I will take care of it.